src/test/java/com/google/javascript/clutz/DeclarationGeneratorTests.java

```diff
@@ -97,8 +97,11 @@ static String getExternFileNameOrNull(String testFileName) {
     File[] testFiles = getPackagePath().toFile().listFiles(filter);
     // Partial files live in 'partial' dir and run implicitly with the --partialInput option on.
     File[] testPartialFiles = getPackagePath().resolve("partial").toFile().listFiles(filter);
+    File[] testMultifilePartailFiles =
```

Comment:
Add a comment describing the purpose of multifile partial, similar to the one above. Call out specifically that each .js creates its own golden test *and* we do a singleton test on top.

Reply:
Added!